gpudirect-tcpxo/nccl-tcpxo-installer.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nccl-tcpxo-installer
+  namespace: kube-system
+  labels:
+    k8s-app: nccl-tcpxo-installer
+spec:
+  selector:
+    matchLabels:
+      k8s-app: nccl-tcpxo-installer
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: nccl-tcpxo-installer
+        k8s-app: nccl-tcpxo-installer
+    spec:
+      priorityClassName: system-node-critical
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: cloud.google.com/gke-accelerator
+                    operator: In
+                    values:
+                      - nvidia-h100-80gb
+      tolerations:
+        - operator: "Exists"
+      hostNetwork: true
+      hostPID: true
+      volumes:
+        - name: var-lib
+          hostPath:
+            path: /var/lib
+        - name: tcpxo
+          hostPath:
+            path: /var/lib/tcpxo
+        - name: library-dir-host
+          hostPath:
+            path: /home/kubernetes/bin
+      initContainers:
+        - image: "ubuntu"
```

Comment:
Can this be a distroless image such as `gke.gcr.io/gke-distroless/bash:latest`?

Reply:
distroless image (e.g. gke.gcr.io/gke-distroless/bash:latest) can't run`nsenter` command, nsenter is part of the `util-linux`, which are generally not included in distroless images.